src/etc/inc/openvpn.tls-verify.php

```diff
@@ -72,6 +74,37 @@
 	}
 }
 
+preg_match('/\/var\/etc\/openvpn\/server(\d+)\/config\.ovpn/', $_GET['config'], $current_vpnid);
+foreach ($config['openvpn']['openvpn-server'] as $ovpns) {
+	if (($ovpns['vpnid'] == $current_vpnid['1']) && ($ovpns['ocspcheck'] == 'yes')) {
+		$capath = "/var/etc/openvpn/server{$ovpns['vpnid']}/ca/";
+		$ca = lookup_ca($ovpns['caref']);
+		$cert_contents = base64_decode($ca['crt']);
+        	$cert_details = openssl_x509_parse($cert_contents);
+	        $issuer = $capath . $cert_details['hash'] . ".0";
+		$serial = $_GET['serial'];
+		$status = exec("/usr/bin/openssl ocsp -issuer {$issuer} 
```

Comment:
Each of these variables needs `escapeshellarg()`

Reply:
done